packages/examples/src/relay/pure-relay/Feedback.stories.tsx

```diff
@@ -177,7 +180,9 @@ export const LikeFailure: Story = {
         { mockClientData: true },
       );
     });
-    const likeButton = await container.findByRole("button", { name: "Like" });
+    const likeButton = await container.findByRole("button", {
+      name: '<mock-value-for-field-"likeLabel">', // No mock resolvers available here, so we use the generated value
```

Comment:
So let's add them, I think just specifying resolver for ViewData in line 178, would be enought

Reply:
Perfect! :D I tried to add them to this place first, but that's not allowed when `enableQueuedMockResolvers` is false.

```ts
 parameters: {
    novaEnvironment: {
      enableQueuedMockResolvers: false,
    },
  } satisfies WithNovaEnvironment<FeedbackStoryRelayQuery, TypeMap>,
  ```
  
  I somehow missed the resolvers further down in the test. Thanks!